site/content/includes/installation/install-gateway-api-experimental-features.md

```diff
@@ -0,0 +1,27 @@
+---
+docs: "DOCS-000"
+---
+
+Installing Gateway API resources from the experimental channel includes everything in the standard release channel plus additional experimental resources and fields.
+NGINX Gateway Fabric currently supports a subset of the additional features provided by the experimental channel. In order
```

Comment:
Thoughts on linking to https://docs.nginx.com/nginx-gateway-fabric/overview/gateway-api-compatibility/ to show which additional features NGF supports?

Reply:
I like it!